core/src/test/java/io/aiven/kafka/tieredstorage/manifest/SegmentIndexesV1BuilderTest.java

```diff
@@ -39,7 +39,8 @@ void shouldFailWhenBuildingWithNotEnoughIndexes() {
             .add(RemoteStorageManager.IndexType.TIMESTAMP, 1)
             .build())
             .isInstanceOf(IllegalStateException.class)
-            .hasMessage("Not enough indexes have been added. At least 4 required");
+            .hasMessage("Not enough indexes have been added; at least 4 required. "
+                + "Indexes included: [TIMESTAMP, OFFSET]");
```

Comment:
This one is flaky: the order of index types in the exception message is not guaranteed.

Reply:
Good catch! Turning it into a sorted list, have a look at the last commit.